format.py

```diff
@@ -44,17 +46,25 @@ def write(types, string_dict, ty, tree, out):
   # Check the AST conforms to the IDL.
   tycheck.TypeChecker(types).check_any(ty, tree)
 
-  # At this point, we are done with verifications. We may start writing the output.
   out.write(MAGIC_HEADER)
   out.write(FORMAT_VERSION)
 
-  # Collect the local strings and write the string table
+  # The content is brotli-compressed
+  local_strings_before_brotli = io.BytesIO()
```

Comment:
can this be named `uncompressed_local_string` or something?
`before` sounds like position inside file.